documentation/modules/ROOT/pages/setup.adoc

```diff
@@ -81,8 +81,8 @@ The `work` folder in `$TUTORIAL_HOME` can be used to download the demo applicati
 
 This tutorial was developed and tested with:
 
-- Knative `v0.3.0`
-- minikube `v0.34.1`
+- Knative `v0.4.0`
```

Comment:
You also need to update knative-version in documentation/modules/ROOT/pages/_attributes.adoc .

But, there's a wrinkle - we shouldn't just assume build/serving/eventing are all the exact same version. Serving is on 0.4.1, for example - patch releases get cut for each automatically based on need. So, we should separate out the versions of build/serving/eventing and be explicit with each. That could wait for a subsequent PR, but noting it here for now.

Reply:
@bbrowning  thanks for the head up, in earlier versions all were same, since its getting changed will have it updated.